writer.py

```diff
@@ -1,4 +1,8 @@
 
+#open and write to the file located at path
```

Comment:
Remember that in python - unlike java - docstrings are not just comments before a function, they're bare strings inside a function; see https://en.wikipedia.org/wiki/Docstring#Python and https://www.python.org/dev/peps/pep-0257/